install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -60,6 +55,9 @@ spec:
         {{- toYaml . | nindent 8 }}
         {{- end }}
     spec:
+      # Ensure priority scheduling
+      priorityClassName: "system-node-critical"
+
```

Comment:
is it already defined below ?

https://github.com/cilium/cilium/blob/f0fbba16a07d84d0ffc8c64bac3b5442e49a37ea/install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml#L443

Reply:
Thanks I missed that.